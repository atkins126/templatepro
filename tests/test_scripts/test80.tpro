DataSets
{{loop(customers)}}
Code: {{customers.Code}} - Name: {{customers.Name}}
{{exit}}
{{endloop}}

List of objects
{{loop(objects)}}
Prop1: {{objects.Prop1}} {{value2}}
---
{{endloop}}