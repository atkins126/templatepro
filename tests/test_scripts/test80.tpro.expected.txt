DataSets
Code: 1 - Name: Ford
